flow-typed/npm/jest_v22.x.x_with_custom_matchers.js

```diff
@@ -110,21 +110,29 @@ type JestPromiseType = {
   resolves: JestExpectType
 };
 
+/**
+ * Jest allows functions and classes to be used as test names in test() and
+ * describe()
+ */
+type JestTestName = string | Function;
```

Comment:
Wut? Really?

Reply:
Just copy pasted this from flow/flow-typed repo :D